### DISCUSSION TITLE
a serious security issue

### DISCUSSION THREAD

**Discussion by xerox-xeon** [projectdiscovery/cloudlist#450] [Q&A]
When I run the cloudlist program, the Server Guard of my server issues an alarm, and the cloudlist process connects to this IP, 144.126.254.30
What caused this?

**Comment by ehsandeep** [comment]
@lifeblood not a security issue or bug, it's version check call to [pdtm](https://github.com/projectdiscovery/pdtm) api to check current installed version of cloudlist or any pd tool + to enable the support self-update option.

```
https://api.pdtm.sh/api/v1/tools/cloudlist
```

```
host api.pdtm.sh
api.pdtm.sh has address 144.126.254.30
```
